packs/data/stolen-fate-bestiary.db/raskus.json

```diff
@@ -1095,11 +1095,20 @@
                             {
                                 "or": [
                                     "target:condition:blinded",
-                                    "target:condition:immobilized"
+                                    "target:condition:immobilized",
+                                    "opportunistic-attack"
                                 ]
                             }
                         ],
                         "selector": "strike-damage"
+                    },
+                    {
+                        "domain": "damage",
+                        "key": "RollOption",
+                        "label": "Target has reduction to Speed",
```

Comment:
Consider localizing the label.

Reply:
We can localize later, need this in now.